consensus/src/round_manager.rs

```diff
@@ -995,6 +979,22 @@ impl RoundManager {
                     BlockStage::ORDER_VOTED,
                 );
             }
+            // generate and multicast randomness share for the fast path
```

Comment:
Earlier, we were multicasting randomness shares when receiving the block proposal. Now, we are doing it when broadcasting order votes. Why add one hop delay here?